### ISSUE TITLE
Ensure documentation is accessible

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#16675]
**This is a Feature Request**

**What would you like to be added**
Look at the set of documentation (initially, the English localization).
Review this against [ARIA](https://www.w3.org/TR/wai-aria/) (Accessible Rich Internet Applications) guidelines / specifications.
Make changes to ensure that the documentation is accessible.

**Why is this needed**
> Education and collaboration are vital to the future of the CNCF ecosystem and it is imperative to us that everyone in the community that wants to participate feels welcome to do so regardless of gender, gender identity, sexual orientation, disability, race, ethnicity, age, religion or economic status.

 &mdash; CNCF

**Comments**
Treat this as an overarching / epic issue.
After identifying room for improvement, log a specific issue about making that improvement, and mention this issue in the description of the new issue.

**Comment by aimeeu** [comment]
Good idea! Would it be possible to prepend "Umbrella Issue: " to the title? Easier to find Epic-like issues that way ;-p

**Comment by sftim** [comment]
/retitle Umbrella issue: ensure documentation is accessible

**Comment by aimeeu** [comment]
This issue may belong under this umbrella: Remove insensitive wordings from the docs like just - easy or - simple  https://github.com/kubernetes/website/issues/17070
